packages/amplify_authenticator/test/ui/tab_view_test.dart

```diff
@@ -67,191 +80,202 @@ void main() {
+            labelColor: Colors.amber,
+          ),
+          indicatorColor: Colors.pink,
+        ),
+        darkTheme: ThemeData.dark().copyWith(
+          tabBarTheme: const TabBarTheme(
+            labelColor: Colors.amber,
+          ),
+          indicatorColor: Colors.pink,
+        ),
+        useAmplifyTheme: false,
+        amplifyConfig: config,
+      );
 
   // The Amplify theme.
-  final amplify = buildTabView(
-    useAmplifyTheme: true,
-  );
+  Widget amplify(String config) => buildTabView(
+        useAmplifyTheme: true,
+        amplifyConfig: config,
+      );
 
   group('AuthenticatorTabView', () {
     setUp(() {
       binding.window.clearPlatformBrightnessTestValue();
     });
 
-    group('Light Mode', () {
-      setUp(() {
-        binding.window.platformBrightnessTestValue = Brightness.light;
-      });
-
-      // Tests the default Material theme.
-      testWidgets('(default)', (WidgetTester tester) async {
-        await tester.pumpWidget(defaultMaterial);
-        await tester.pumpAndSettle();
-
-        await expectGoldenMatches(
-          tabViewFinder,
-          'tab_view_light_default.png',
-        );
-      });
-
-      // Tests the default high contrast theme.
-      testWidgets('(high contrast)', (WidgetTester tester) async {
-        await tester.pumpWidget(highContrast);
-        await tester.pumpAndSettle();
-
-        await expectGoldenMatches(
-          tabViewFinder,
-          'tab_view_light_high_contrast.png',
-        );
-      });
-
-      // Tests a custom Material swatch.
-      testWidgets('(swatch)', (WidgetTester tester) async {
-        await tester.pumpWidget(customSwatch);
-        await tester.pumpAndSettle();
-
-        await expectGoldenMatches(
-          tabViewFinder,
-          'tab_view_light_swatch.png',
-        );
-      });
-
-      // Tests a manual override of the tab color.
-      testWidgets('(override)', (WidgetTester tester) async {
-        await tester.pumpWidget(override);
-        await tester.pumpAndSettle();
-
-        await expectGoldenMatches(
-          tabViewFinder,
-          'tab_view_light_override.png',
-        );
-      });
-
-      // Tests the Amplify theme.
-      testWidgets('(amplify)', (WidgetTester tester) async {
-        await tester.pumpWidget(amplify);
-        await tester.pumpAndSettle();
-
-        await expectGoldenMatches(
-          tabViewFinder,
-          'tab_view_light_amplify.png',
-        );
-      });
-    });
-
-    group('Dark Mode', () {
-      setUp(() {
-        binding.window.platformBrightnessTestValue = Brightness.dark;
-      });
-
-      // Tests the default Material theme (dark mode).
-      testWidgets('(default)', (WidgetTester tester) async {
-        await tester.pumpWidget(defaultMaterial);
-        await tester.pumpAndSettle();
-
-        await expectGoldenMatches(
-          tabViewFinder,
-          'tab_view_dark_default.png',
-        );
+    for (var config in configs.entries) {
```

Comment:
This diff here appears large, but it is mostly whitespace. See https://github.com/aws-amplify/amplify-flutter/pull/1195/files?diff=split&w=1 to ignore whitespace changes.